codegen/build.gradle.kts

```diff
@@ -205,13 +205,13 @@ subprojects {
         tasks["test"].finalizedBy(tasks["jacocoTestReport"])
 
         // Configure jacoco to generate an HTML report.
-        tasks.jacocoTestReport {
-            reports {
-                xml.isEnabled = false
-                csv.isEnabled = false
-                html.destination = file("$buildDir/reports/jacoco")
-            }
-        }
+//        tasks.jacocoTestReport {
+//            reports {
+//                xml.isEnabled = false
+//                csv.isEnabled = false
+//                html.destination = file("$buildDir/reports/jacoco")
+//            }
+//        }
```

Comment:
Question: Why no test report?

Reply:
I had commented this out as my local instance of gradle has issues with this. Forgot to remove it, but it's done now.